app/players/elves/druid.py

```diff
@@ -0,0 +1,19 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(
+            self,
+            nickname: str,
+            musical_instrument: str,
+            favourite_spell: str) -> None:
+        super().__init__(nickname, musical_instrument)
+        self.favourite_spell = favourite_spell
+
+    def get_rating(self) -> int:
+        return len(self.favourite_spell)
```

Comment:
Ensure that the `get_rating` method accesses the `_favourite_spell` attribute if you change its name to be protected.